samples/dnn/fcn_semsegm.cpp

```diff
@@ -50,8 +50,9 @@ static void colorizeSegmentation(const Mat &score, const vector<cv::Vec3b> &colo
     const int cols = score.size[3];
     const int chns = score.size[1];
 
-    cv::Mat maxCl(rows, cols, CV_8UC1);
-    cv::Mat maxVal(rows, cols, CV_32FC1);
+    cv::Mat maxCl=cv::Mat::zeros(rows, cols, CV_8UC1);
+    cv::Mat maxVal=Mat(rows, cols, CV_32FC1, cv::Scalar(-FLT_MAX));
+    
```

Comment:
1. Avoid calling of unnecessary assignment operator (copy/move constructor):
```.diff
cv::Mat maxVal=Mat(rows, cols, CV_32FC1, cv::Scalar(-FLT_MAX));
cv::Mat maxVal(rows, cols, CV_32FC1, cv::Scalar(-FLT_MAX));
```

2. Please remove [spaces](http://pullrequest.opencv.org/buildbot/builders/precommit_docs/builds/12928) from line 55.

3. Squash commits into one (to eliminate fixup commits from Git history), use `git rebase -i HEAD~3` (or HEAD~4) command. After done this, use `git push` with "force" flag.

Reply:
Done, hope I did it correct, because I did not used rebase before.

Reply:
Unfortunately commits from other patches are appear here. You should do something like this:
```
$ git rebase -i bc348eb8ab051d464e36998935d3716bda446882
... edit ('d' for unrelated commits, 'pick' your first commit, 'f' for your next commits) ...
d 1d5fc195f async bugfix (or remove this line)
d 5eebd68a1 fixed non c++11 warnings. (or remove this line)
pick f62bb8dcf Matrices in colorizeSegmentation of  fcn_semsegm.cpp were not initialized.
f f762f35ab Matrices in colorizeSegmentation were not initialized.
f 56a747037 Matrices in colorizeSegmentation were not initialized.
... save & exit editor ...
$ git push -f ...
```